components/pickerbutton/stories/pickerbutton.stories.js

```diff
@@ -153,3 +126,23 @@ export default {
 
 export const Default = Template.bind({});
 Default.args = {};
+
+export const WithLabel = Template.bind({});
+WithLabel.args = {
+	label: "Select",
+};
+
+export const Disabled = Template.bind({});
+Disabled.args = {
+	isDisabled: true
+};
```

Comment:
Noticing we can toggle focused and open when it is disabled is true, do we want to not allow this?